docs: DIS-133 and DIS-134 plus copyediting (#1439)
ai-dynamo/dynamo: docs/architecture/distributed_runtime.md

- Python: `/lib/bindings/python/examples/`. We also provide a complete example of using `DistributedRuntime` for communication and Dynamo's LLM library for prompt templates and (de)tokenization to deploy a vllm-based service. Please refer to `lib/bindings/python/examples/hello_world/server_vllm.py` for details.

```{note}

> Building a vLLM docker image for ARM machines currently involves building vLLM from source, which is known to have performance issues require extensive system RAM; see [vLLM Issue 8878](https://github.com/vllm-project/vllm/issues/8878).

Reviewer: "Performance issue" sounds weird in context of build, makes it sound like resulting install of vLLM will have performance issues at runtime.

```suggestion
Building a vLLM docker image for ARM machines currently involves building vLLM from source, which is known to be slow and requires extensive system RAM; see [vLLM Issue 8878](https://github.com/vllm-project/vllm/issues/8878).
```
